src/components/FlowRunTimeline.vue

```diff
@@ -193,8 +233,21 @@
 <style>
 .flow-run-timeline { @apply
   flex
+  h-[320px]
   overflow-hidden
   relative
+  outline-none
+}
+
+.flow-run-timeline--fullscreen { @apply
+  h-screen
+  w-full
+  absolute
+  top-0
+  left-0
+  z-50
```

Comment:
ah good eye! Looks like 20 will allow it to still lay over the left menu, on mobile it's working nice as well:
<img width="380" alt="image" src="https://user-images.githubusercontent.com/6776415/219044361-3e24780e-8816-451e-b5b3-cc1b7b868ae9.png">

I checked z-index of the modal, looks like it's at `z-10`? Which is probably ok actually, since it's in a different stacking context at the bottom of the page.

Reply:
Hm we should probably bump that up but I can't see a scenario where a modal overlays this content.